orbital_mouse/orbital_mouse.c

```diff
@@ -261,6 +263,23 @@ bool process_record_orbital_mouse(uint16_t keycode, keyrecord_t* record) {
           select_mouse_button(keycode - OM_SEL1);
         }
         return false;
+      // Check if cardinal snapping is desired
+      case OM_CS_U:
+        state.angle = 0 << 8; //16*0
```

Comment:
Nit: Guard these cases with `if (record->event.pressed) { ... }` so that the angle snaps only on press events. Otherwise, it snaps again on key release, which seems unintended.